DataFormats/StdDictionaries/src/classes_others.h

```diff
@@ -28,10 +28,6 @@ namespace DataFormats_StdDictionaries {
   std::random_access_iterator_tag randaccitertag;
   std::set<int> dummy19;
   std::set<std::basic_string<char> > dummy20;
-  __gnu_cxx::new_allocator<char> dummy21;
```

Comment:
@davidlt - any memory about why you added these (only 4 years ago..)
https://github.com/cms-sw/cmssw/commit/1235397718d013d26a4d7a935b54ff5faf410d4e